aspnetcore/tutorials/grpc/grpc-start/samples/GrpcStart/GrpcGreeter.Server/Services/GreeterService.cs

```diff
@@ -1,20 +1,39 @@
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+//#define ServerCallContext_ONLY
+
 using Greet;
 using Grpc.Core;
+using System.Threading.Tasks;
 
 namespace GrpcGreeter
 {
+#if ServerCallContext_ONLY
+    #region snippet
     public class GreeterService : Greeter.GreeterBase
     {
-        public override Task<HelloReply> SayHello(HelloRequest request, ServerCallContext context)
+        public override Task<HelloReply> 
```

Comment:
What's up with the formatting to separate this into separate lines?

Reply:
That gets rid of the horizontal scroll bar on tablets. Make the widest browser window with no left or right TOC, and you get typical tablet size.

Reply:
BTW, this is WIP and not ready for review :-)